npm_and_yarn/lib/dependabot/npm_and_yarn/helpers.rb

```diff
@@ -1,21 +1,30 @@
 # typed: true
 # frozen_string_literal: true
 
+require "dependabot/dependency"
+require "dependabot/file_parsers"
+require "dependabot/file_parsers/base"
+require "sorbet-runtime"
+
 module Dependabot
   module NpmAndYarn
     module Helpers
+      extend T::Sig
+
       YARN_PATH_NOT_FOUND =
         /^.*(?<error>The "yarn-path" option has been set \(in [^)]+\), but the specified location doesn't exist)/
 
+      sig { params(lockfile: DependencyFile).returns(Integer) }
       def self.npm_version_numeric(lockfile)
```

Comment:
I did a bunch of these to understand what I needed to do in other files, but didn't want to bloat this PR more by finishing it up just yet.